test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py

```diff
@@ -173,14 +176,14 @@ def test_tfs_model(
 @pytest.mark.model("unknown_model")
 @pytest.mark.neuron_test
 def test_tfs_neuron_model(
-    model_name, tfs_neuron_model, sagemaker_regions, image_uri, instance_type, accelerator_type
+    model_name, sagemaker_regions, image_uri, instance_type, accelerator_type
 ):
     input_data = {"instances": [[[[1, 10], [2, 20]]]]}
     util.create_and_invoke_endpoint_helper(
```

Comment:
`invoke_sm_endpoint_helper_function` can directly be used here. We can get rid of `create_and_invoke_endpoint_helper` entirely